public/js/public.js

```diff
@@ -438,6 +438,12 @@ const OrderForCustomer = {
       window.onbeforeunload = function() {
         return "You are attempting to leave this page. When you leave you exit 'Place Order for Customer' mode. Are you sure you want to exit this page?"
       }
+      $('[name="woocommerce_checkout_place_order"]').click(function(_) {
+        window.onbeforeunload = ""
+      })
+      $('#cfw-customer-info-action [data-tab="#cfw-payment-method').click(function(_) {
```

Comment:
I've never seen the `function(_)` that underscrore, that's interesting

Reply:
Yeah, it's my habit from typescript. When i have to add param in function but don't have use for that param I add underscore.

Reply:
Oh cool, so it's an unused variable, nice